subsys/bluetooth/shell/iso.c

```diff
@@ -46,7 +46,7 @@ static struct bt_iso_chan_ops iso_ops = {
 };
 
 static struct bt_iso_chan_qos iso_qos = {
-	.sca = 0x07,
+	.sca = BT_GAP_SCA_UNKNOWN,
```

Comment:
You are now defaulting to 0x00 instead of 0x07, intentional?

Reply:
Yes :) 

Per the core spec, if the SCA of the peripherals are unknown, the value 0 (BT_GAP_SCA_UNKNOWN) shall be used.